winml/adapter/TensorBaseHelpers.h

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
```

Comment:
what is this file for?

Reply:
great catch !

---
In reply to: [354039267](https://github.com/microsoft/onnxruntime/pull/2534#discussion_r354039267) [](ancestors = 354039267)